### PR TITLE
add isScrollable helper and tests :)

### PR DESCRIPTION
**Pull request by leekevinyg**
Added an isScrollable helper to determine if an element is scrollable by checking if the ```scrollHeight > clientHeight``` and checking that the computed style is configured for scrolling i.e., has ```overflowY === 'scroll'``` or ```overflowY === 'auto'```

It looks like there is an issue starting xfvb that is preventing the travis check from going through :(

### CODE REVIEW

File: tests/index.js [hunk 1]
```diff
@@ -228,3 +229,28 @@ describe('getAllElementTypes', () => {
...
+  it('should return false if element is not scrollable', () => {
+    expect(isScrollable(container)).to.be.false; // eslint-disable-line no-unused-expressions
+  });
+
+  it('should return true if element is scrollable', () => {
+    container.style = 'overflow-y:scroll;height:400px';
+    let containerElement = document.createElement('div');
+    containerElement.style = 'height:800px';
+    container.appendChild(containerElement);
+    expect(isScrollable(container)).to.be.true; // eslint-disable-line no-unused-expressions
```
Comment (sljavi) [inline tests/index.js]: I think it should be `expect(isScrollable(container)).to.equal(true)` instead

File: src/element-scrollable.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export const checkScrollable = (element) => {
```
Comment (sljavi) [inline src/element-scrollable.js]: this is great

File: tests/index.js [hunk 3]
```diff
@@ -228,3 +229,28 @@ describe('getAllElementTypes', () => {
...
+    container = document.createElement('div');
+    document.body.appendChild(container);
+  });
+
+  afterEach(() => {
+    document.body.removeChild(container);
+  });
+
+  it('should return false if element is not scrollable', () => {
+    expect(isScrollable(container)).to.be.false; // eslint-disable-line no-unused-expressions
```
[the remark this answers is not in the document]
Comment (leekevinyg) [reply tests/index.js]: Fixed! I'm not sure if ```to.be``` and ```to.equal``` has a difference when it comes to primitive boolean values, but changed anyway :)

File: tests/index.js (hunk 1 above)
Comment (leekevinyg) [reply tests/index.js]: Fixed

Comment (sljavi) [reply tests/index.js]: You're totally right. I found it weird that the assert can be executed without the `(` and `)`at the end, but it's what the official doc says. 
¯\_(ツ)_/¯
https://www.chaijs.com/api/bdd/#method_true
sorry for requesting the unnecessary change

Comment (leekevinyg) [reply tests/index.js]: No worries! Both work and I learned more about chai functionality :) 

It looks like I have not able to merge this PR until you approve it. Let me know if there's any other changes that need to be pushed, will be happy to fix them up!

**Pull request merged**